modules/_commands.js

```diff
@@ -88,11 +88,13 @@ module.exports = {
   events: {
     message: function (bot, nick, to, text, message) {
       let controlChar = bot.config.get('irc.controlChar')
+      const spaceNotControl = new RegExp(' [^' + controlChar + ']')
```

Comment:
what happens if I set the control character to `\`?

Reply:
Inserting a literal backslash into the regex would work for any character that isn't `d`, `s`, `w`, or something else special. Requiring a `controlChar` not in `[A-Za-z0-9]` seems like a reasonable restriction anyway.

Or I could regex the regex...